apps/location_service/config/config.exs

```diff
@@ -6,11 +6,25 @@ use Mix.Config
 config :location_service, :http_pool, :google_http_pool
 
 if Mix.env() == :prod do
+  config :location_service,
```

Comment:
No special reason, I think it was the outcome of a weird mix of (1) following conventions in some of the other dotcom apps, and (2) setting it up so that it would default to different values between different environments. 

In this setup basically it'll default to AWS unless you opt-in to use Google, or (on `:prod`) default to Google unless you opt-in to use AWS-- I was interested in avoiding any mishaps where an environment was using AWS before we were quite ready, but also wanted to make it easy to develop using AWS without necessarily having to remember to use the environment variable. That said it's all made up, e.g. these env vars aren't set elsewhere or doing anything right now 🙃

Reply:
I don't think this is going to work the way you expect. These files are evaluated at compile time, so the `if` statement will be checked and converted to `:google`. You might want to use `config/runtime.exs` instead, which can check the environment variable directly and set the appropriate values.

Reply:
@paulswartz Welp. We're on a version of Elixir that predates the addition of `config/runtime.exs` (1.11)

Reply:
Elixir 1.10 has `config/releases.exs`: https://hexdocs.pm/mix/1.10.0/Mix.Tasks.Release.html